main.js

```diff
@@ -0,0 +1,25 @@
+const menu = document.querySelector('.menu');
+const menuOption = document.querySelector('.menu_opt');
+const closeMenu = document.querySelector('.close_menu');
+
+menu.addEventListener('click', () => {
+  menuOption.classList.remove('open');
+  menuOption.style.display = 'block';
+  menuOption.style.height = '100vh';
+  menu.style.visibility = 'hidden';
+  closeMenu.style.visibility = 'visible';
+});
+
+closeMenu.addEventListener('click', () => {
+  menuOption.classList.toggle('open');
+  menuOption.style.height = '0';
+  menu.style.visibility = 'visible';
+  closeMenu.style.visibility = 'hidden';
+});
+
+menuOption.addEventListener('click', () => {
+  closeMenu.style.visibility = 'hidden';
+  menuOption.style.display = 'none';
+  menuOption.style.height = '0';
+  menu.style.visibility = 'visible';
+});
```

Comment:
- [ ] On your mobile menu it is possible to accidentally click on the 'purple space' below the mobile options and close the menu(see gif) when it should be possible to close the menu only by clicking on the 'X' button or the menu options. This can be a bad user experience. Please make your mobile menu close functionality works only by clicking on the 'X' button or the menu options.